runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/fn/control/ProcessRemoteBundleOperation.java

```diff
@@ -54,13 +54,16 @@
   private final StageBundleFactory stageBundleFactory;
   private static final OutputReceiver[] EMPTY_RECEIVER_ARRAY = new OutputReceiver[0];
   private final Map<String, OutputReceiver> outputReceiverMap;
+
+  @SuppressWarnings("UnnecessaryAnonymousClass")
```

Comment:
I was going to request we go ahead and resolve these, but I'm not really clear on how to do it. It seems like we'd need to do some refactoring. So I think this is fine for now.